tools/test-npm.sh

```diff
@@ -29,10 +29,10 @@ export npm_config_prefix="npm-prefix"
 export npm_config_tmp="npm-tmp"
 
 # install npm devDependencies and run npm's tests
-
-../$NODE cli.js install --ignore-scripts
-../$NODE cli.js run-script test-legacy
-../$NODE cli.js run-script test
+NODEPATH="$(../$NODE -p 'require("path").resolve("..")')"
+PATH="$NODEPATH:$PATH" ../$NODE cli.js install --ignore-scripts
+PATH="$NODEPATH:$PATH" ../$NODE test/run.js
+PATH="$NODEPATH:$PATH" ../$NODE cli.js run-script tap -- "test/tap/*.js"
```

Comment:
Ok one thing here: could you do `export PATH="$NODEPATH:$PATH"` above these three lines? To my understanding it should give the same effect.